regression/contracts/assigns_enforce_18/test.desc

```diff
@@ -0,0 +1,21 @@
+CORE
+main.c
+--enforce-all-contracts _ --pointer-primitive-check
+^EXIT=10$
+^SIGNAL=0$
+^\[bar.\d+\] line \d+ Check that \*b is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that a is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that yp is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that z is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that \*xp is assignable: SUCCESS$
+^\[foo.\d+\] line \d+ Check that y is assignable: FAILURE$
+^.* 1 of \d+ failed \(\d+ iteration.*\)$
+^VERIFICATION FAILED$
+--
+^\[foo.pointer\_primitives.\d+\] line \d+ deallocated dynamic object in POINTER_OBJECT\(tmp\_cc\$\d+\): FAILURE$
+^.* 8 of \d+ failed (\d+ iterations)$
```

Comment:
I don't understand this negative regex. Why are we checking `8` out of "anything"?
We should list out the ones of interest that are not supposed to pass / fail explicitly, just like the assertion above.

Reply:
I ran this regression test with our `develop` branch and included the failed checks in the negative regex.